runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -307,6 +307,16 @@ public static class CorfuRuntimeParameters extends RuntimeParameters {
 
         // endregion
 
+        /**
+         * Default client name in case the client chooses to remain anonymous.
+         */
+        String clientName = "";
```

Comment:
we need a _string_ to separate out the different processes as part of syslog integration otherwise we cannot distinguish the server logs from client logs which clientId is not.